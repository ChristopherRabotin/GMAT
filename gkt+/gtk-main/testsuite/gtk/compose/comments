<Multi_key> <a> : "a" # just an a
<Multi_key> <b> : "#" # not an a
# this line is ignored
<Multi_key> <c> : "a" # wild and wilder: "@!$#
