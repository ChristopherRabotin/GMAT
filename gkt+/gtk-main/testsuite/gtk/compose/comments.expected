# n_sequences: 3
# max_seq_len: 2
# n_index_size: 1
# data_size: 9
# n_chars: 0
<Uff20> <U61> : "a" # U61
<Uff20> <U62> : "#" # U23
<Uff20> <U63> : "a" # U61
